app/experimenter/nimbus-ui/src/hooks/useCommonForm/index.test.tsx

```diff
@@ -144,5 +155,42 @@ describe("hooks/useCommonForm", () => {
         expect(screen.queryByTestId(name)).toBeInTheDocument();
       });
     });
+
+    it("FormBranch", () => {
```

Comment:
I find it *vaguely* squicky to test `FormBranch` specifically here, rather than a more generic test subject that exercises useCommonForm-specific features. Kind of tangles the two up. But, we can worry about that if/when this gets used in more cases.